examples/02_decoding/plot_haxby_stimuli.py

```diff
@@ -27,8 +26,8 @@
     for i in range(48):
         plt.subplot(6, 8, i + 1)
         try:
-            plt.imshow(imread(file_names[i]), cmap=plt.cm.gray)
-        except:
+            plt.imshow(plt.imread(file_names[i]), cmap=plt.cm.gray)
+        except Exception:
```

Comment:
We should make it a more specific exception, FileNotFoundError  or something only catching missing file condition.